packages/firebase_ai/firebase_ai/lib/src/api.dart

```diff
@@ -679,8 +679,8 @@ enum FinishReason {
   /// Convert to json format
   String toJson() => _jsonString;
 
-  // ignore: unused_element
-  static FinishReason _parseValue(Object jsonObject) {
+  // ignore: public_member_api_docs
+  static FinishReason parseValue(Object jsonObject) {
```

Comment:
Any reason we don't document this when parseCitationMetadata is properly documented?